spacemacs/packages.el

```diff
@@ -136,6 +137,35 @@ which require an initialization must be listed explicitly in the list.")
       (setq ace-jump-mode-scope 'global)
       (evil-leader/set-key "`" 'ace-jump-mode-pop-mark))))
 
+(defun spacemacs/init-ace-link ()
+  (use-package ace-link
+    :defer t
+    :init (ace-link-setup-default)
```

Comment:
Indeed, `:commands spacemacs/ace-buffer-links` should be sufficient, the autoloaded commands should work without requiring to add them explicitly. You can use `(macroexpand '(use-package ..))` to see what is the result.